security/calife/Portfile

```diff
@@ -1,51 +1,53 @@
-PortSystem		1.0
-
-name			calife
-version			3.0.5
-categories		security
-license         GPL-2
-maintainers		keltia.net:roberto openmaintainer
-description		Provides super user privileges to specific users
-long_description	Calife is a small utility close to sudo(8) but \
-			lightweight and it preserves the caller's environment \
-			(shell aliases, PATH and so on).
-homepage		https://www.keltia.net/programs/calife
-master_sites	https://assets.keltia.net/calife
-platforms		darwin
-
-use_xz		yes
-checksums \
-	rmd160 c5b68045102773c86e65b87c3a99879e4e360188 \
-	sha256 beda669de952875699ad99b27781eebd9393e6becd41961f87278f6682f3b903
-use_configure		yes
-
-configure.args-append	--with-etcdir=${prefix}/etc
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                calife
+version             3.0.6
+categories          security
+license             GPL-2
+maintainers         keltia.net:roberto openmaintainer
+description         Provides super user privileges to specific users
+long_description    Calife is a small utility close to sudo(8) \
+                    but lightweight, and it preserves the caller’s environment \
+                    (shell aliases, PATH and so on).
+homepage            https://www.keltia.net/programs/calife
+master_sites        https://assets.keltia.net/calife
+
+checksums           rmd160  92fe1a21d3df98120ce66b696503cffe90472d52 \
+                    sha256  f09b5c73439bab14a7ff1438b8f22a8da6216a5caf0ef8467f85467b2217357e \
+                    size    98768
+use_xz              yes
+
+use_configure       yes
+
+configure.args-append --with-etcdir=${prefix}/etc
 
 destroot {
-  xinstall -m 755 -d ${destroot}${prefix}/bin
-  xinstall -m 755 -d ${destroot}${prefix}/share/calife
-  xinstall -m 755 -d ${destroot}${prefix}/share/man/man1
-  xinstall -m 755 -d ${destroot}${prefix}/share/man/man5
-
-  xinstall -m 4755 ${worksrcpath}/calife \
-			${destroot}${prefix}/bin/calife
-  xinstall -m 444 ${worksrcpath}/pam/macosx \
-			${destroot}${prefix}/share/calife/pam.d-calife
-  xinstall -m 600 ${worksrcpath}/calife.auth-dist \
-			${destroot}${prefix}/etc
-  xinstall -m 644 ${worksrcpath}/calife.1 \
-			${destroot}${prefix}/share/man/man1
-  xinstall -m 644 ${worksrcpath}/calife.auth.5 \
-			${destroot}${prefix}/share/man/man5
+    xinstall -m 755 -d ${destroot}${prefix}/bin
+    xinstall -m 755 -d ${destroot}${prefix}/share/calife
+    xinstall -m 755 -d ${destroot}${prefix}/share/man/man1
+    xinstall -m 755 -d ${destroot}${prefix}/share/man/man5
```

Comment:
are the "bin" and "man?" directories not made by default already?

Reply:
This has been there in the Portfile. I can try removing it, if you suggest.